optuna/cli.py

```diff
@@ -910,6 +937,12 @@ def _add_common_arguments(parser: ArgumentParser) -> ArgumentParser:
             "Also can be specified via OPTUNA_STORAGE environment variable."
         ),
     )
+    parser.add_argument(
+        "--storage-class",
+        help="Storage class hint (e.g. JournalFileStorage)",
+        type=str,
```

Comment:
This is not a strong opinion, but what do you think of the `choices` argument to enumerate the acceptable storage classes?  It could potentially help users understand its usage more readily. On the other hand,  we would then need to maintain the list along with the implementation of `_get_storage`.

Reply:
Sounds reasonable! I will update the PR.